script.js

```diff
@@ -0,0 +1,67 @@
+const movie = [
+    {
+    title: "The Shawshank Redemption",
+  },
+  {
+    title: "The Godfather",
+  },
+  {title: "The Dark Knight",},
+  {
+    title: "12 Angry Men",
+  },
+  {
+    title: "The Lord Of The Rings",
+  },
+  {
+    title: "Pulp Fiction",
+  },
+  {
+    title: "The Good, the Bad and the Ugly",
+  },
+  {
+    title: "Forrest Gump",
+  }
+]
+
+for (let j = 0; j < movie.length; j++){
+
+    console.log(movie[j]);
+    
+    }
+
+   
+    let movieList = [
+        "The Shawshank Redemption",
+        "The Godfather",
+        "The Dark Knight",
+        "12 Angry Men",
+        "The Lord Of The Rings",
+        "Pulp Fiction",
+        "The Good, the Bad and the Ugly",
+        "Forrest Gump"
+    ]
+
+    for (let j = 0; j < movieList.length; j++){
+
+        console.log(movieList[j]); 
+        }
+
+    let counter = 0;
+
+        for (let elem of movieList) {
```

Comment:
I see what you wanted to do here, to count the occurrence of words you know. This is not scalable though, cause you cannot add an `if` for every word. Using Google, you would have found a way simpler solution, if Github wasn't so annoying and stupid that day, and if you had more time, I am sure of it!

Reply:
Matter of fact, try finding such a solution on Google, i.e. `javascript count number of words in string`